addons/sourcemod/scripting/vsh/bosses/boss_whiteranger.sp

```diff
@@ -0,0 +1,380 @@
+#define RANGER_MODEL 		"models/player/vsh_rewrite/uber_ranger/uber_ranger.mdl"
+#define RANGER_THEME 		"vsh_rewrite/uber_ranger/uberrangers_music.mp3"
+#define RANGER_RAGESOUND 	"mvm/mvm_tele_deliver.wav"
+
+static int g_iUberRangerPrussianPickelhaube;
+static int g_iUberRangerBlightedBeak;
+
+static bool g_bUberRangerPlayerWasSummoned[TF_MAXPLAYERS+1];
+
+static char g_strUberRangerRoundStart[][] = {
+	"vo/medic_battlecry05.mp3"
+};
+
+static char g_strUberRangerWin[][] = {
+	"vo/medic_specialcompleted12.mp3",
+	"vo/taunts/medic/medic_taunt_kill_22.mp3",
+	"vo/medic_autocappedcontrolpoint01.mp3"
+};
+
+static char g_strUberRangerLose[][] = {
+	"vo/medic_paincrticialdeath01.mp3",
+	"vo/medic_paincrticialdeath02.mp3",
+	"vo/medic_paincrticialdeath03.mp3",
+	"vo/medic_paincrticialdeath04.mp3",
+};
+
+static char g_strUberRangerRage[][] = {
+	"vo/medic_specialcompleted04.mp3",
+	"vo/medic_specialcompleted05.mp3",
+	"vo/medic_specialcompleted06.mp3"
+};
+
+static char g_strUberRangerJump[][] = {
+	"vo/medic_go05.mp3",
+	"vo/medic_cheers05.mp3"
+};
+
+static char g_strUberRangerLastMan[][] = {
+	"vo/taunts/medic_taunt_kill_08.mp3"
+};
+
+static char g_strUberRangerBackStabbed[][] = {
+	"vo/medic_autodejectedtie01.mp3",
+	"vo/medic_sf12_badmagic10.mp3", 
+	"vo/medic_taunts11.mp3"
+};
+
+methodmap CUberRanger < SaxtonHaleBase
+{
+	public CUberRanger(CUberRanger boss)
+	{
+		CBraveJump abilityJump = boss.CallFunction("CreateAbility", "CBraveJump");
+		abilityJump.iJumpChargeBuild = 1;
+		
+		CRageAddCond rageCond = boss.CallFunction("CreateAbility", "CRageAddCond");
+		rageCond.flRageCondDuration = 6.0;
+		rageCond.flRageCondSuperRageMultiplier = 1.333;	//This is 7.998 seconds, close enough
+		rageCond.AddCond(TFCond_UberchargedCanteen);
+		
+		boss.iBaseHealth = 650;
+		boss.iHealthPerPlayer = 550;
+		boss.nClass = TFClass_Medic;
+		boss.iMaxRageDamage = 2500;
+		boss.bCanBeHealed = true;
+	}
+	
+	public void GetBossName(char[] sName, int length)
+	{
+		strcopy(sName, length, "White Über Ranger");
+	}
+	
+	public void GetBossInfo(char[] sInfo, int length)
+	{
+		StrCat(sInfo, length, "\nHealth: Low");
```

Comment:
Go for `Very Low`, there bosses like Gentle Spy with more health, but also said health is `Low`